staging/velero/values.yaml

```diff
@@ -38,11 +38,15 @@ metrics:
   enabled: false
   scrapeInterval: 30s
 
+  service:
+    # labels:
+    #   key: value
+
   # Pod annotations for Prometheus
-  podAnnotations:
```

Comment:
We're planning on providing our staging changes to upstream charts in time. This will require some iterations on the current implementation in the future, so it would be helpful for future readers to add a comment here providing an explanation and the context from the PR as to why this is now commented out.

Reply:
+1